app/src/main/java/com/duckduckgo/app/browser/WebDataManager.kt

```diff
@@ -64,6 +70,11 @@ class WebViewDataManager @Inject constructor(
         webView.clearFormData()
     }
 
+    override suspend fun deleteWebViewDirectory() {
```

Comment:
I find it confusing that this method is called `clearWebViewDirectory` but only clears _some_ of the web view directory. Furthermore it has knowledge of other clearing behavior e.g it knows it doesn't need to clear the cookies and storage because other things clear those.

It could be renamed to better capture what it is doing or take the excluded items as a parameter. If the former, it should also be private as it would still contain knowledge of an "overall clearing strategy" and so shouldn't be part of the public API (that should be the top-level method that does the overall clearing).

Reply:
Good points, and 👍 for the pragmatic suggestion to not block on it.

Can discuss it more here: https://app.asana.com/0/414730916066338/1127379946457993

Reply:
Given that, is this PR approved to go?